### PR TITLE
Dashboard Queries, Charts, Lists

### PR DESCRIPTION
**Pull request by dthyresson**
Demonstrates how to use Prisma Raw Query and GraphQL sdl to have some general mechanisms to start querying data for metrics.

Note:

* numbers are numbers
* dates are dates
:)

This implements a few dashboard charts as well.

Still some rework and improvements, but would like to get merged so don't stray too far off main.

### CODE REVIEW

File: api/src/services/metrics/metrics.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+export const metricSqlStatements: QueryResolvers['metricSqlStatements'] =
+  async () => {
+    const typeId = 'SQL'
+    const attributeKey = 'db.statement'
+
+    return (await spansByAttributeKeyAndType(
+      typeId,
+      attributeKey
+    )) as SpansByAttributeKeyAndType[]
```
Comment (Tobbe) [inline api/src/services/metrics/metrics.ts:47]: If you move this type cast to the `spansByAttributeKeyAndType` function at least you'd only have to do it once

**Comment by dthyresson** [comment]
Added performance queries to make charts!

**Comment by dthyresson** [comment]
Implemented some performance charts 
image

**Comment by dthyresson** [comment]
I also reinstated the Schema and ERD pages. Reworked the sidebar, paths, and routes to reorganize.

Stubbed out basic fencing for sql statements and gqwl operation lists. Needs pagination (as do spans, traces as these get huge).

**Comment by dthyresson** [comment]
I want to reorg all icons in a tax with purposeful names so pages, comports import "DatabaseIcon" and that can be a stack a circle or whatever -- to make it easier to group, name, rename and change consistent icons across app.

**Comment by dthyresson** [comment]
I reworked the icons. There is still something a bit odd about the sql statements and operations query data that I have to audit as my query sometimes does not return what I'd expect. for example I see inserts into contacts when I didn't do that, but the timestamp shows it was recently done.

**Comment by dthyresson** [comment]
> for example I see inserts into contacts when I didn't do that, but the timestamp shows it was recently done.

My bad. I was not joining on the implicit Prisma join tables, so it was cartesian joining exploding on all attributes:

```sql

  -- here we use the Prisma implicit join table to join attributes A to spans B
  JOIN _OTelTraceAttributeToOTelTraceSpan atos on atos.A = a.id and atos.B = s.id
```

fixes

**Comment by Tobbe** [comment]
> I would like to change up the icons again though as having one place for icons is nice

Done: https://github.com/redwoodjs/studio/commit/26342eab7ad9d7e96562ef5c12f9e460bf664d9d

**Pull request merged**